aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,84 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## January 4 - January 21, 2022
+
+### "Big Things" progress updates
+Here are the latest developments regarding roadmap items:
+- **Tag** - Completed guidance for Tags in [Figma](https://www.figma.com/file/8zrInotU4JK9LWP4WyZy25/HPE-Design-System-Tags?node-id=289%3A3190) and published on [Design System site](https://design-system.hpe.design/components/tag?q=tag)
+- **Navigation** 
+   - Completed research for [Ascending Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2382%3A13827)
+   - Completed ideations for [Ascending Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2725%3A18556)
+- **Button** - Updated font [color on default button active states from default text to text-strong.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component)
+- **Page Layouts** 
+   - Completed Dashboard template for a [2 column layout.](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2231%3A17992)
+   - Shared status update with the Design System Council on January 20, 2022.
+   - Completed documentation around the definition, scope and anatomy of Page Layouts
+   - Created guidelines for the Page Container level, including defining 3 types of content widths: Wide, Narrow and Full
+   - Defined new breakpoints and page margins for each breakpoint
+   - Currently working on creating Page Layout templates, starting with 2 column, 3 column and multi-column Dashboards
+
+### Stay current with the latest changes
+- **Voice and Tone** - Updates in progress for Voice & Tone page.
+- **404 Page** - Designed a new 404 page for our Design System site.
+- **Accessibility** - Updates in progress for Accessibility page to provide more resources and guidance.
+- **SideBar** - Removed sidebar from examples and documentation, this component is going to be deprecated by the end of the year. 
+
+### Latest Releases
+- **Grommet v2.20.0** - Release includes the NEW Tag component along with enhancements to Calendar, Carousel, DateInput, Form, Video; fixes applied to Anchor, Button, Calendar, Clock, Select, Text, and TextInput. Full details about this release are available in the [Release Notes.](https://github.com/grommet/grommet/releases/tag/v2.20.0)
+
+## December 24 - January 3, 2022 Happy Holidays 🎉
+
+## December 20 - 23, 2021
+
+### "Big Things" progress updates
```

Comment:
Same as above, soft opinion but should we call this just "Roadmap Updates" OR "Big Things Updates" -- I think having "progress" and "updates" is redundant.